src/routes/SafeAppLandingPage/components/SafeAppsDetails.tsx

```diff
@@ -19,7 +20,13 @@ type SafeAppDetailsTypes = {
 
 const SafeAppDetails = ({ iconUrl, name, description, availableChains }: SafeAppDetailsTypes): ReactElement => {
   const showAvailableChains = availableChains?.length > 0
-  const { isModalVisible: isLoaded, isSafeAppInDefaultList } = useSecurityFeedbackModal()
+  const { isLoading: isSafeAppListLoading, getSafeApp } = useAppList()
+  const { getAppUrl } = useSafeAppUrl()
+  const url = getAppUrl()
```

Comment:
`url` is a part of `safeAppDetails` variable which is passed to this component, you don't need to call `getAppUrl` here again

Reply:
Nice!, added a param instead using the hook